backendImpl/src/test/java/com/virtuslab/gitmachete/backend/integration/StatusAndDiscoverIntegrationTestSuite.java

```diff
@@ -214,13 +209,13 @@ private void printCommonParts(IManagedBranchSnapshot branch, List<INonRootManage
       sb.append("  ");
       sb.append(customAnnotation.get());
     }
-    val syncToRemote = branch.getSyncToRemoteStatus();
+    val syncToRemote = branch.getRelationToRemote();
```

Comment:
Rename the local variable

Reply:
Btw, `val` (short for _value_) means two things:
1. the type is automatically guessed based on what you assign to the variable (you don't need to specify it yourself)
2. the variable is **final** (can't be ever reassigned), so it's not really a variable but right more of a constant